docker/wrapper.py

```diff
@@ -12,37 +12,131 @@
 import gzip
 from bd2k.util.exceptions import require
 from toil.lib.bioio import addLoggingOptions, setLoggingFromOptions
+import time
+import virtualenv
+import re
+from itertools import chain
+
+from shutil import copyfile
+#import ConfigParser
+
+import socket
 
 logging.basicConfig(level=logging.INFO)
 log = logging.getLogger()
 
-
 def call_pipeline(mount, args):
-    work_dir = os.path.join(mount, 'Toil-RNAseq-' + str(uuid4()))
+
+    #Activate the virtual environment where the toil rnaseq pipeline is installed
+    activate_this = '/opt/rnaseq-pipeline/toil_venv/bin/activate_this.py'
+    execfile(activate_this, dict(__file__=activate_this))
+
+    if args.auto_scale:
+        #Run the meoso-master process
+        #Run it in the background by using Popen instead of check_call 
+        #https://stackoverflow.com/questions/32577071/execute-subprocess-in-background
```

Comment:
Great comments (Although PEP8 is one space after the `#`)

Reply:
added space